File: OCH_Tao/ABC/A/027.py
L = input().split()
if L.count(L[0]) == 3:
  print(L[0])
elif L.count(L[0]) == 1:
  print(L[0])
else:
  if L[0] == L[1]:
    print(L[2])
  else:
    print(L[1])

File: OCH_Tao/ABC/A/027.js
const input = ((require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n"))[0].split(" ")).map(str => parseInt(str, 10));
if (input[0] === input[1] && input[0] === input[2]) {
  console.log(input[0]);
} else if (input[1] === input[2]) {
  console.log(input[0]);
} else if (input[0] === input[2]) {
  console.log(input[1]);
} else {
  console.log(input[2]);
}